openapi/tests/openapi_integration/helpers/collection_setup.py

```diff
@@ -49,7 +49,7 @@ def basic_collection_setup(collection_name='test_collection', on_disk_payload=Fa
                 {
                     "id": 1,
                     "vector": [0.05, 0.61, 0.76, 0.74],
-                    "payload": {"city": "Berlin"}
```

Comment:
The `values_count` change is just an example. What I meant to say is without having the test data representing all cases, some change may break another one and never fail a test since they're not existing in the same collection.

I was already reverted the test changes to their previous state and added a new collection with the diverse payloads per @generall's request. However, I am not happy with this solution for the reasons mentioned above. I'd rather add to the old test points.